packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -1,13 +1,55 @@
-# Memory leaks tracking with Dart DevTools
+# Dart DevTools memory leak tracker
 
 This page and functionality are under construction. See https://github.com/flutter/devtools/issues/3951.
 
 [self-link](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md)
 
+
 ## Understand concepts
 
+### Glossary
+
+**GC**: garbage collection
+
+**Memory Leak**: progressive usage of more and more memory by an application
+
+**Memory Bloat**: use of more memory than is necessary for optimal performance
+
+**Object's Retaining Path**: path from the object to a root object that prevents the object from being GCed
+
+
+### Leak Types
+
+To detect memory leaks, the tool uses the fact that, with proper memory management, a dart object disposal and GC events should happen sequentially, close to each other.
+
+By disposal and GC events, the tool detects different types of leaks:
+
+**Not disposed, but GCed (not-disposed)**: a disposable object was GCed, without being disposed. This means that the object's disposable content was allocating memory, after the object became not needed. To fix the leak, invoke `dispose()` when the object is not needed.
```

Comment:
nit: change this to "GCed, but not disposed" to match the sentence structure of "Disposed, but not GCed". Makes it a little easier to understand by putting the positive first

Reply:
it is intentional as disposal should happen first